extensions/wikia/PageHeader/templates/PageHeaderController_categories.php

```diff
@@ -0,0 +1,29 @@
+<?php if ( $categories->hasVisibleCategories() ): ?>
+	<div class="page-header__categories">
+		<span class="page-header__categories-in" data-tracking="categories-top-in"><?= $categories->inCategoriesText ?>:</span>
+		<div class="page-header__categories-links">
+			<?= $categories->visibleCategoriesHTML ?>
```

Comment:
You can always set a come in the same line as link :) and I also agree that it's definitely better to have links generated in a template. I won't be stubborn here, but me no likey ;) if more of you agrees it's ok to be done this way - I'll click "approve".